Fixed issue #7

nisrulz opened the pull request.

nisrulz committed:
fixes issue #7

kartikarora commented:
@nisrulz This seems to have codacy issues. Could fix them and send in a new commit? We'll have a look at it before merging.

nisrulz committed:
[*] updated
- support lib to v25.1.0
- build tools version to v25.0.1
- code inspected and corrected for issues
- fixed memory leak when referencing the edit text of activity inside fragment
